src/main/java/org/terasology/climateConditions/BodyTemperatureComponent.java

```diff
@@ -0,0 +1,15 @@
+// Copyright 2020 The Terasology Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+package org.terasology.climateConditions;
+
+import org.terasology.entitySystem.Component;
+import org.terasology.network.Replicate;
+
+public class BodyTemperatureComponent implements Component {
```

Comment:
I think this should also get some JavaDoc, stating what the range and interpretation of the `current` temperature is. You may also state which system is (supposed to be) managing this component (not intended for direct write access).